src/lib/Microsoft.Health.Fhir.Ingest/Service/MeasurementFhirImportService.cs

```diff
@@ -41,10 +42,10 @@ public async Task ProcessStreamAsync(Stream data, string templateDefinition, ITe
             var template = BuildTemplate(templateDefinition, log);
             var measurementGroups = await ParseAsync(data, log).ConfigureAwait(false);
 
-            await ProcessMeasurementGroups(measurementGroups, template, log).ConfigureAwait(false);
+            await ProcessMeasurementGroups(measurementGroups, template, log, CancellationToken.None).ConfigureAwait(false);

```

Comment:
Is this meant to be `CancelationToken.None`? Should it be a cancellation token that is passed into `ProcessStreamAsync`?

Reply:
This particular method is the legacy azure functions code path which we'll be deprecating soon so I didn't spend much time on it and just wanted to keep the current state behavior